templates/haskell/package.yaml

```diff
@@ -0,0 +1,139 @@
+# Use this file as a reference when you create package.yaml
+# for your new package.
+# If you have multiple packages, you may wish to extract common
+# definitions into a separate file.
+# Why we need this template:
+# • harmless default extensions
+# • a good set of warnings
+# • metadata fields that are better be set
+# • some tricks (e. g. RTS options)
+name:                myproject
+# version is 0 by default to indicate that nothing is ready
+version:             0
+# adjust/add more authors and maintainers if necessary
+author:              Serokell
+maintainer:          Serokell <hi@serokell.io>
+# other metadata fields we usually set:
+# • copyright
```

Comment:
why not include some default fields?
```
copyright: "{YEAR} Serokell"
license: MPL-2.0
homepage: "github.com/serokell/{PROJECT}"
bug-reports: "{AUTHOR NAME}@serokell.io"
```
Then a newcomer doesn't have to guess

Reply:
I am just afraid that people may blindly copy-paste these default values into their `package.yaml` files even when they are invalid. For example, in my team the copyright owner is usually not Serokell, but our customer, also the repos are often not in `github.com/serokell`. However, I agree that having examples is a good idea. I will think what to do here and will expand it.